### DISCUSSION TITLE
How can I deploy a Angular project on Hostinger?

### DISCUSSION THREAD

**Discussion by devpedro-dias** [community/community#139544] [Programming Help] [answered]
How can I deploy a Angular project on Hostinger?

I'm having a problem with the deployment on Hostinger, where I import my repository in the File Manager but when I go to deploy, the site returns a 403 Forbidden. I've already tried to solve it by following the videos that suggest this error and it didn't work. I searched for how to deploy Angular projects on Youtube and still couldn't. Could someone help me?

Project: https://github.com/devpedro-dias/scx-website

**Comment by emma009** [comment, marked as answer]
i think you need to do

`ng build --configuration production ` or npm build

then when you done...move the content of www direct to ur hosting root folder

when you done create .htaccess file within the root folder and add the following code, to avoid 404 error

RewriteEngine on

```
# Don't rewrite files or directories
RewriteCond %{REQUEST_FILENAME} -f [OR]  
RewriteCond %{REQUEST_FILENAME} -d  
RewriteRule ^ - [L]

# Rewrite everything else to index.html to allow html5 state links
RewriteRule ^ index.html [L]  
```

**Reply by devpedro-dias** [reply]
I ran ```ng build --configuration production``` , and it generated the ```dist/test/browser``` folder. I placed it in the public_html folder in the File Manager on Hostinger. After that, do I add all the other folders (components, pages...)? And in the case of generating the .htaccess file, would I just create and paste what you sent me? I apologize for the questions, but I'm a layman and this is the first time I've tried to deploy something on Hostinger.

![image](https://github.com/user-attachments/assets/1a5c689e-1425-4df2-be94-4d9147d7e51e)

Browser folder:

![image](https://github.com/user-attachments/assets/9f6782fb-a87c-4eab-9dc3-38681eb98a14)

**Reply by emma009** [reply]
thats because your profile name i think is test...so move all files and folders inside test folder to your root folder within your hosting account..thanks

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @devpedro-dias ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
